tracer/samples/AutomaticTraceIdInjection/RunSampleWithAutomaticInstrumentation.sh

```diff
@@ -0,0 +1,40 @@
+## Set the project you want to test
+PROJECT=Log4NetExample
+APP_PATH=./bin/Debug/net7.0
+
+## Setup automatic instrumentation
+## cf https://www.nuget.org/packages/Datadog.Trace.Bundle#readme-body-tab
+## Note that the agent must be configured if you want to receive the traces
+
+## .NET Core
+export CORECLR_ENABLE_PROFILING=1
+export CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+## Replace this path by the appropriate one depending on your system (cf https://www.nuget.org/packages/Datadog.Trace.Bundle#readme-body-tab)
+export CORECLR_PROFILER_PATH=$APP_PATH/datadog/osx/Datadog.Trace.ClrProfiler.Native.dylib
+export DD_DOTNET_TRACER_HOME=$APP_PATH/datadog
+
+## .NET Framework
+# export COR_ENABLE_PROFILING=1
+# export COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+## Replace this path by the appropriate one depending on your system (cf https://www.nuget.org/packages/Datadog.Trace.Bundle#readme-body-tab)
+# export COR_PROFILER_PATH=$APP_PATH\datadog\win-x64\Datadog.Trace.ClrProfiler.Native.dll
+# export DD_DOTNET_TRACER_HOME=<APP_DIRECTORY>/datadog
```

Comment:
_Arguably_ there's not much point in having Windows configuration in a bash script. Maybe this should be a powershell script instead (given that runs on all platforms) 😛

Reply:
Made this into a powershell script `BuildAndRunSample.ps1` along with some additional configuration